source/projects/contact_manager.markdown

```diff
@@ -2276,8 +2275,8 @@ To get this to pass I changed my create method to this:
 ```ruby
 def create
   data = request.env['omniauth.auth']
-  user = User.where(:provider => data['provider'], :uid => data['uid'], :name => data['info']['name']).first_or_create
-  render :nothing => true
+  user = User.where(providerL data['provider'], uid: data['uid'], name: data['info']['name']).first_or_create
```

Comment:
@applegrain `provider:` instead of `providerL` :cop: 

Overall this looks really good :sparkles: